test/ComposerGpgVerifyTest/VerifyTest.php

```diff
@@ -0,0 +1,445 @@
+<?php
+
+declare(strict_types=1);
+
+namespace ComposerGpgVerifyTest;
+
+use Composer\Composer;
+use Composer\Config;
+use Composer\Script\Event;
+use Composer\Script\ScriptEvents;
+use ComposerGpgVerify\Verify;
+use PHPUnit\Framework\TestCase;
+use Symfony\Component\Process\Process;
+
+/**
+ * @covers \ComposerGpgVerify\Verify
+ */
+final class VerifyTest extends TestCase
```

Comment:
if the `--points-at HEAD` were to go missing from your code; it'd introduce a subtle backdoor into the verification. Adding tests at least makes this a louder change.